### PR TITLE
Valid Code For Build

### PR DESCRIPTION
**Pull request by lblemail**
Add CMakeLists and main func make code valid

### CODE REVIEW

File: src.cpp [hunk 1]
```diff
@@ -1,2 +1,11 @@
 // This is the test source file
 #include "header.h"
+
+using std::cout;
+using std::endl;
+
+int main(int argc, char** argv)
+{
+  cout << "Test Git" << endl;
```
Comment (divinel) [inline src.cpp:9]: Here has conflict

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+project(git_test)
+
+add_executable(git_test src.cpp)
```
Comment (divinel) [inline CMakeLists.txt:3]: Good we can build using cmake now

**Review by divinel — CHANGES_REQUESTED** [review]
Fix conflict before the code can be merged

File: src.cpp (hunk 1 above)
Comment (lblemail) [reply src.cpp:9]: Ok, I will fix this **conflict**

File: src.cpp [hunk 3]
```diff
@@ -1,7 +1,11 @@
 // This is the test source file
 #include "header.h"
 
+using std::cout;
+using std::endl;
+
 int main(int argc, char** argv)
 {
+  cout << "Test Git" << endl;
```
Comment (divinel) [inline src.cpp:9]: good, no conflict any more

**Comment by lblemail** [comment]
Conflict *Fixed*!

**Comment by divinel** [comment]
Ok, I accept this change and will rebase the change to master branch

**Review by divinel — APPROVED** [review]

**Review by divinel — CHANGES_REQUESTED** [review]
Add some more info in **README** please

**Comment by lblemail** [comment]
OK, I will add some README Info

**Comment by divinel** [comment]
That will be nice

File: README.md [hunk 4]
```diff
@@ -1,2 +1,4 @@
 # git_test
 Test Git Workflow
+
+Here we add multiple commits to test pull request
```
Comment (lblemail) [inline README.md:4]: Here add the purpose of this project

Comment (divinel) [reply README.md:4]: Nice

**Pull request merged**